### PR TITLE
Disabling installer test execution for now.

### PR DESCRIPTION
**Pull request by abayer**
The tests have a tendency to hang badly and don't pass in the first
place, so disabling them for now.

cc @reviewbybees

### CODE REVIEW

**Comment by svanoort** [comment]
:-1: - Rather than disabling them completely, we should run them to see which are passing and failing, but handle the errors and allow the process to continue.  We still get value from seeing which tests are failing over time.  *Especially* if jenkins suddenly fails to start.  

If we disable tests, then we enter a state where we stop caring entirely what's failing, and there's no incentive to fix the actual underlying issues.

@abayer  The periodic hangs need to be fixed ASAP, this is the first I have heard of it.  Have you submitted a JIRA for them with diagnostic info or logged this somewhere appropriate?

**Comment by abayer** [comment]
I'm solely aiming this at the Jenkins 2.0 build for the moment - the build is *always* red, so it's hard to tell if anything is meaningful at this point. I'm going to set up a separate installer test job for now.

And no, no JIRA because things have hung at inconvenient enough times that I've had to kill them with fire. =)

**Comment by svanoort** [comment]
After a out-of-band discussion, :+1: with following actions agreed upon:

1. Separate job created by @abayer that is fired for each build, so that we still have visibility into the failures and can identify critical regressions
2. Once installertests are in a state where they will return green except for critical failures, they will be included in the main path for packaging.  Noncritical failures may be segregated into a separate testset - this may be assisted by xUnit compatible output.

**Comment by abayer** [comment]
ping for @reviewbybees

**Comment by svanoort** [comment]
:bee: as before

File: Jenkinsfile [hunk 1]
```diff
@@ -122,34 +122,37 @@ timestampedNode('docker') {
 
 }
 
-stage "Package testing"
+// Installer testing currently disabled - it hangs periodically and doesn't actually *pass* anyway, so holding off on running it for now.
+if (false) {
```
Comment (jtnord) [inline Jenkinsfile]: you could have avoided the whole indentation with `if (runTests && false)`

**Comment by jtnord** [comment]
:bee:

File: Jenkinsfile (hunk 1 above)
Comment (abayer) [inline Jenkinsfile]: ...well, yeah. Sorry. =)

**Pull request merged**